### DISCUSSION TITLE
We should unify and upgrade "Font Awesome" to the latest version 6.5.x

### DISCUSSION THREAD

**Discussion by aescanes** [monkeytypegame/monkeytype#5428] [Ideas] [answered]
Working on a small change (updating the Twitter logo with the X logo), I found that we have a custom solution for "Font Awesome" and and different versions are being used (5.15.4, 5.13.0, etc). We can unify all using the latest version of "Font Awesome" 6.5.x

**Comment by fehmer** [comment, marked as answer]
I started working on the version upgrade in #5423

**Reply by aescanes** [reply]
Great. I didn't see that, I'm sorry.
